tests/x11regressions/gnomecase/nautilus_open_ftp.pm

```diff
@@ -24,6 +24,11 @@ sub run() {
     send_key "ret";
     assert_screen 'nautilus-ftp-suse-com', 20;
 
+    if (get_var('SP2ORLATER')) {
```

Comment:
good that you ask. The whole version checking has not been there as is for long. For now you are right: The effect will be the same. The variable "SP2ORLATER" is there because needle matching can only check for variables in boolean context and openQA can not check like "MATCH if $version >= 12SP2". Within the tests however we can use the IMHO more explicit and easier to read `version_at_least('12-SP2')`.